docker/package/wizard_structure.py

```diff
@@ -715,13 +715,13 @@ def import_key(self, key_mode_query, ledger_app=None):
                         f"Waiting for funds to arrive... (Ctrl + C to choose another option)."
                     )
                     try:
-                        # dry-run delegate registration until it succeeds
                         while True:
                             result = get_proc_output(
                                 f"sudo -u tezos {suppress_warning_text} octez-client {tezos_client_options} "
-                                f"register key {baker_alias} as delegate --dry-run"
+                                f"register key {baker_alias} as delegate"
                             )
                             if result.returncode == 0:
+                                print(result.stdout.decode("utf8"))
```

Comment:
Why add this? :thinking:

Reply:
Because otherwise user chose `generate-fresh-key` option won't get `octez-client` output with baker registering information